packages/openai-compatible/README.md

```diff
@@ -0,0 +1,77 @@
+# AI SDK - OpenAI Compatible Provider
+
+This package provides a foundation for implementing providers that expose an OpenAI-compatible API.
+
+The primary [OpenAI provider](../openai/README.md) is more feature-rich, including OpenAI-specific experimental and legacy features. This package offers a lighter-weight alternative focused on core OpenAI-compatible functionality.
+
+## Setup
+
+The provider is available in the `@ai-sdk/openai-compatible` module. You can install it with
+
+```bash
+npm i @ai-sdk/openai-compatible
+```
+
+## Provider Instance
+
+You can import the provider creation method `createOpenAICompatible` from `@ai-sdk/openai-compatible`:
+
+```ts
+import { createOpenAICompatible } from '@ai-sdk/openai-compatible';
+```
+
+## Example
```

Comment:
There should be a section on creating a custom provider leveraging this package. I'm planning to add that once we have the TogetherAI one done.